.vitepress/config.mts

```diff
@@ -35,6 +35,339 @@ function nav(): DefaultTheme.NavItem[] {
       link: '/reference/torii-endpoints',
       activeMatch: '/reference/',
     },
+    {
+      text: 'Cookbook',
+      link: '/cookbook/',
+      activeMatch: '/cookbook/',
+    },
+  ]
+}
+
+function sidebarCookbook(): DefaultTheme.SidebarItem[] {
+  return [
+    {
+      text: 'Access Control',
+      collapsed: true,
+      items: [
+        {
+          text: 'Permission Tokens',
+          collapsed: true,
+          items: [
+            {
+              text: 'Grant Permissions'
+            },
+            {
+              text: 'Revoke Permissions'
+            },
+            {
+              text: 'Mint Permission Tokens'
+            },
+            {
+              text: 'Burn Permission Tokens'
+            },
+          ],
+        },
+        {
+          text: 'Roles',
+          collapsed: true,
+          items: [
+            {
+              text: 'Register a Role',
+            },
+            {
+              text: 'Grant a Role',
+            },
+            {
+              text: 'Revoke a Role',
+            },
+          ],
+        },
+        {
+          text: 'Executors (Permission Validators)',
```

Comment:
```suggestion
          text: 'Executors',
```

Executor is more than a permission validator, and I think it deserves it's own section

Reply:
no, I would remove the section `Executors` out of `Access Control` and make it a top-level section. It should look like this:
```
        {
          text: 'Executors',
          collapsed: true,
          items: [
            {
              text: 'Write Executor',
            },
            {
              text: 'Update Executor',
            },
            {
              text: 'Define custom permission tokens',
            },
          ],
        }
```